<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Publications | Olly King</title>
  <link rel="shortcut icon" href="/images/favicon.png"/>

  <!-- Social media cards -->
  <!-- Twitter -->
  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:site" content="@_OllyK">
  <meta name="twitter:creator" content="@_OllyK">
  <!-- OpenGraph -->
  <meta property="og:type" content="website" />
  <meta property="og:title" content="Publications | Olly King" />
  <meta property="og:description" content="A list of my academic publications" />
  <meta propergty="og:image" content="https://www.ollyking.com/images/thumbnail/publications.png" />
  <meta property="og:url" content="https://www.ollyking.com/publications/index.html" />
  <!--<meta property="og:site_name" content="Olly King" />-->

  <!-- Bootstrap -->
  <link rel="stylesheet" href="/pkg/bootstrap-5.1/bootstrap.min.css">
  <!-- FontAwsome -->
  <link rel="stylesheet" href="/pkg/fontawesome/css/all.css">
  <!-- Academicons -->
  <link rel="stylesheet" href="/pkg/academicons-1.9.1/css/academicons.min.css">
  <!-- KaTeX -->
  <link rel="stylesheet" href="/pkg/katex-0.15.3/katex.min.css" integrity="sha384-KiWOvVjnN8qwAZbuQyWDIbfCLFhLXNETzBQjA/92pIowpC0d2O3nppDGQVgwd2nB">

  <!-- Main CSS stylesheet -->
  <link rel="stylesheet" href="/css/style.css">

  <!-- Plausible analytics for anonymous usage statistics -->
  <script defer data-domain="leouieda.com" src="https://plausible.io/js/plausible.js"></script>
</head>

<body>
  <header style="">
    <nav class="navbar navbar-expand-lg navbar-dark" aria-labelledby="primary-navigation">
      <div class="container align-middle">
        <a class="navbar-brand" href="/">
          OLLY KING
        </a>
        <button class="navbar-toggler" type="button"
                data-bs-toggle="collapse" data-bs-target="#navbarNav"
                aria-controls="navbarNav" aria-expanded="false"
                aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse align-middle" id="navbarNav">
        <div class="navbar-collapse align-middle" id="navbarNav">
          <ul class="navbar-nav ms-lg-auto flex-row flex-wrap">
              <li class="nav-item col-6 col-lg-auto">
                
  
  
  
  <a class="nav-link"  href="/">Home</a>
              </li>
              <li class="nav-item col-6 col-lg-auto">
                
  
  
  
  <a class="nav-link"  href="/about/">About</a>
              </li>
              <li class="nav-item col-6 col-lg-auto">
                
  
  
  
    
    
  
  <a class="nav-link active" aria-current="page" href="/publications/">Publications</a>
              </li>
          </ul>
        </div>
      </div>
    </nav>
      <div class="banner-background" style="background-image: url(/images/diffractometer_hd_wide.jpg); background-position: top left;">
        <div class="overlay">
        </div>
        <section class="container text-center banner">
            <h1 class="display-3">
              Publications
            </h1>
            <p class="fs-4">
              A list of my academic publications
            </p>
        </section>
      </div>
  </header>

  <main>
      <section class="container page-content overflow-hidden wide-page">
        <h2 class="mb-4">
Protein Structures in the RCSB Protein Data Bank
</h2>
<button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-99"
      aria-expanded="false" aria-controls="collapse-99">
    See the table <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
</button>
<div id="collapse-99" class="collapse mt-2 overflow-hidden">
<p> These are the novel protein structures that I've contributed to through doing one or a combination of these things: expressing, purifying, crystallising, finding ligands for co-crystallisation, collecting data or solving and refining the structure.
</p>
 <table>
 <tr>
    <th>Class</th>
    <th>Name</th>
    <th>ID</th>
    <th>Name</th>
    <th>ID</th>
    <th>Name</th>
    <th>ID</th>
    <th>Name</th>
    <th>ID</th>
  </tr>
<tr>
  <td>
  <h2 class="fs-4 mb-1">
    Oxygenases
  </h2>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/KDM4A">KDM4A</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3NJY">3NJY</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3PDQ">3PDQ</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3RVH">3RVH</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/4GD4">4GD4</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/5LY1">5LY1</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/HIF1AN">HIF1AN</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3OD4">3OD4</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/4AI8">4AI8</a>
  </h4>
  </td>
  </td>
</tr>
<tr>
  <td>
  <h2 class="fs-4 mb-1">
    Phosphatases
  </h2>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/PTPN11">PTPN11</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3B7O">3B7O</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/PTPRN2">PTPRN2</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2QEP">2QEP</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/PTPN22">PTPN22</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2P6X">2P6X</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/PTPRE">PTPRE</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2JJD">2JJD</a>
  </h4>
  </td>
  </td>
</tr>
<tr>
  <td>
  <h2 class="fs-4 mb-1">
    Kinases
  </h2>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/TTK">TTK</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3CEK">3CEK</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3GFW">3GFW</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3H9F">3H9F</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/TWF2">TWF2</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2VAC">2VAC</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/HASPIN">HASPIN</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2VUW">2VUW</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/CAMK2A">CAMK2A</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2VZ6">2VZ6</a>
  </h4>
  </td>
  </td>
</tr>
<tr>
  <td>
  <h2 class="fs-4 mb-1">
  <p></p>
  </h2>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/CLK3">CLK3</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2WU6">2WU6</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2WU7">2WU7</a>
  </h4>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3RAW">3RAW</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/DYRK1A">DYRK1A</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2VX3">2VX3</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/RPS6KA1">RPS6KA1</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2WNT">2WNT</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/CDPK1">CDPK1</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2WEI">2WEI</a>
  </h4>
  </td>
  </td>
</tr>
<tr>
  <td>
  <h2 class="fs-4 mb-1">
  <p></p>
  </h2>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/GAK">GAK</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/4O38">4O38</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/VRK1">VRK1</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3OP5">3OP5</a>
  </h4>
  </td>
  </td>
</tr>
<tr>
  <td>
  <h2 class="fs-4 mb-1">
    Other
  </h2>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/HIBCH">HIBCH</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3BPT">3BPT</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/CA13">CA13</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/3DA2">3DA2</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/CA8">CA8</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2W2J">2W2J</a>
  </h4>
  </td>
  <td>
   <h2 class="fs-6 mb-1">
    <a target="_blank" href="https://pubchem.ncbi.nlm.nih.gov/gene/RECQL">RECQL</a>
  </h2>
  </td>
   <td>
   <h4 class="fs-6 mb-1 pr-5">
    <a target="_blank" href="https://www.rcsb.org/structure/2V1X">2V1X</a>
  </h4>
  </td>
  </td>
</tr>
</table>
</div>
<h2 class="mb-4">
Research Papers
</h2>
<button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-98"
      aria-expanded="false" aria-controls="collapse-98">
    See the list <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
</button>
<div id="collapse-98" class="collapse mt-2 overflow-hidden">
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    CHiMP - Deep Learning Tools Trained on Protein Crystallisation Micrographs to Enable Automation of Experiments
  </h2>
  <p class="mb-1">
    <span class="text-muted">2024</span>
    |
    King, O.N.F., Levik, K.E., Sandy, J., and Basham, M.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-warning text-dark fw-normal me-1">
        preprint
      </span>
    <b>
    BioRxiv
    </b>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-1"
      aria-expanded="false" aria-controls="collapse-1">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1101/2024.05.22.595345" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-1" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>A group of three deep learning tools, referred to collectively as CHiMP (Crystal Hits in My Plate) were created for analysis of micrographs of protein crystallisation experiments at the Diamond Light Source (DLS) synchrotron, UK. The first tool, a classification network, assigns images into categories relating to experimental outcomes. The other two tools are networks that perform both object detection and instance segmentation, resulting in masks of individual crystals in the first case, and masks of crystallisation droplets in addition to crystals in the second case, allowing positions and sizes of these entities to be recorded. Creation of these tools used transfer learning, where weights from a pre-trained deep learning network were used as a starting point and re-purposed by further training on a relatively small set of data. Two of the tools are now integrated at the VMXi macromolecular crystallography beamline at DLS where they absolve the need for any user input both for monitoring crystallisation experiments and for triggering in situ data collections. The third is being integrated into the XChem fragment-based drug discovery screening platform, also at DLS, to allow automatic targeting of acoustic compound dispensing into crystallisation droplets.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">King, O.N.F., Levik, K.E., Sandy, J., Basham, M. CHiMP - Deep Learning Tools Trained on Protein Crystallisation Micrographs to Enable Automation of Experiments. BioRxiv. 2024. doi: 10.1101/2024.05.22.595345.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Online citizen science with the Zooniverse for analysis of biological volumetric data
  </h2>
  <p class="mb-1">
    <span class="text-muted">2023</span>
    |
    Smith, P., King, O.N.F., Pennington, A., Tun, W.M., Basham, M., Jones, M.L., Collinson, L.M., Darrow, M. C., Spiers, H.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Histochemistry and Cell Biology
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1007/s00418-023-02204-6">10.1007/s00418-023-02204-6</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-2"
      aria-expanded="false" aria-controls="collapse-2">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1007/s00418-023-02204-6" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-2" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Public participation in research, also known as citizen science, is being increasingly adopted for the analysis of biological volumetric data. Researchers working in this domain are applying online citizen science as a scalable distributed data analysis approach, with recent research demonstrating that non-experts can productively contribute to tasks such as the segmentation of organelles in volume electron microscopy data. This, alongside the growing challenge to rapidly process the large amounts of biological volumetric data now routinely produced, means there is increasing interest within the research community to apply online citizen science for the analysis of data in this context. Here, we synthesise core methodological principles and practices for applying citizen science for analysis of biological volumetric data. We collate and share the knowledge and experience of multiple research teams who have applied online citizen science for the analysis of volumetric biological data using the Zooniverse platform ( www.zooniverse.org ). We hope this provides inspiration and practical guidance regarding how contributor effort via online citizen science may be usefully applied in this domain.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Smith, P., King, O.N.F., Pennington, A., Tun, W.M., Basham, M., Jones, M.L., Collinson, L.M., Darrow, M. C., Spiers, H. Online citizen science with the Zooniverse for analysis of biological volumetric data. Histochem Cell Biol. 2023 Jun 7:1-24. doi: 10.1007/s00418-023-02204-6.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1007/s00418-023-02204-6"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Comparison of Methods to Segment Variable-Contrast XCT Images of Methane-Bearing Sand Using U-Nets Trained on Single Dataset Sub-Volumes
  </h2>
  <p class="mb-1">
    <span class="text-muted">2022</span>
    |
    Alvarez-Borges, F.J.; King, O.N.F.; Madhusudhan, B.N.; Connolley, T.; Basham, M.; Ahmed, S.I.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Methane
    </b>
      doi:<a target="_blank" href="https://doi.org/10.3390/methane2010001">10.3390/methane2010001</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-3"
      aria-expanded="false" aria-controls="collapse-3">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.3390/methane2010001" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-3" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Methane (CH4) hydrate dissociation and CH4 release are potential geohazards currently investigated using X-ray computed tomography (XCT). Image segmentation is an important data processing step for this type of research. However, it is often time consuming, computing resource-intensive, operator-dependent, and tailored for each XCT dataset due to differences in greyscale contrast. In this paper, an investigation is carried out using U-Nets, a class of Convolutional Neural Network, to segment synchrotron XCT images of CH4-bearing sand during hydrate formation, and extract porosity and CH4 gas saturation. Three U-Net deployments previously untried for this task are assessed: (1) a bespoke 3D hierarchical method, (2) a 2D multi-label, multi-axis method and (3) RootPainter, a 2D U-Net application with interactive corrections. U-Nets are trained using small, targeted hand-annotated datasets to reduce operator time. It was found that the segmentation accuracy of all three methods surpass mainstream watershed and thresholding techniques. Accuracy slightly reduces in low-contrast data, which affects volume fraction measurements, but errors are small compared with gravimetric methods. Moreover, U-Net models trained on low-contrast images can be used to segment higher-contrast datasets, without further training. This demonstrates model portability, which can expedite the segmentation of large datasets over short timespans.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Alvarez-Borges, F.J.; King, O.N.F.; Madhusudhan, B.N.; Connolley, T.; Basham, M.; Ahmed, S.I. Comparison of Methods to Segment Variable-Contrast XCT Images of Methane-Bearing Sand Using U-Nets Trained on Single Dataset Sub-Volumes. Methane 2023, 2, 1-23. https://doi.org/10.3390/methane2010001</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.3390/methane2010001"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Volume Segmantics: A Python Package for Semantic Segmentation of Volumetric Data Using Pre-trained PyTorch Deep Learning Models
  </h2>
  <p class="mb-1">
    <span class="text-muted">2022</span>
    |
    King, O.N.F, Bellos, D. and Basham, M.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Journal of Open Source Software
    </b>
      doi:<a target="_blank" href="https://doi.org/10.21105/joss.04691">10.21105/joss.04691</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-4"
      aria-expanded="false" aria-controls="collapse-4">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.21105/joss.04691" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-4" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Segmentation of 3-dimensional (3D, volumetric) images is a widely used technique that allows
interpretation and quantification of experimental data collected using a number of techniques
(for example, Computed Tomography (CT), Magnetic Resonance Imaging (MRI), Electron
Tomography (ET)). Although the idea of semantic segmentation is a relatively simple one,
giving each pixel a label that defines what it represents (e.g cranial bone versus brain tissue);
due to the subjective and laborious nature of the manual labelling task coupled with the huge
size of the data (multi-GB files containing billions of pixels) this process is often a bottleneck
in imaging workflows. In recent years, deep learning has brought models capable of fast and
accurate interpretation of image data into the toolbox available to scientists. These models
are often trained on large image datasets that have been annotated at great expense. In many
cases however, scientists working on novel samples and using new imaging techniques do not
yet have access to large stores of annotated data. To overcome this issue, simple software
tools that allow the scientific community to create segmentation models using relatively small
amounts of training data are required. Volume Segmantics is a Python package that provides
a command line interface (CLI) as well as an Application Programming Interface (API) for
training 2-dimensional (2D) PyTorch (Paszke et al., 2019) deep learning models on small
amounts of annotated 3D image data. The package also enables applying these models to
new (often much larger) 3D datasets to speed up the process of semantic segmentation.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">King, O.N.F., Bellos, D. and Basham, M (2022). Volume Segmantics: A Python Package for Semantic Segmentation of Volumetric Data Using Pre-trained PyTorch Deep Learning Models. Journal of Open Source Software, 7(78), 4691, https://doi.org/10.21105/joss.04691</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.21105/joss.04691"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    SuRVoS 2: Accelerating Annotation and Segmentation for Large Volumetric Bioimage Workflows Across Modalities and Scales
  </h2>
  <p class="mb-1">
    <span class="text-muted">2022</span>
    |
    Pennington, A., King, O.N.F., Tun, W.M., Ho, E.M.L., Luengo, I., Darrow, M.C., and Basham, M.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Frontiers in Cell and Developmental Biology
    </b>
      doi:<a target="_blank" href="https://doi.org/10.3389/fcell.2022.842342">10.3389/fcell.2022.842342</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-5"
      aria-expanded="false" aria-controls="collapse-5">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.3389/fcell.2022.842342" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-5" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>As sample preparation and imaging techniques have expanded and improved to include a variety of options for larger sized and numbers of samples, the bottleneck in volumetric imaging is now data analysis. Annotation and segmentation are both common, yet difficult, data analysis tasks which are required to bring meaning to the volumetric data. The SuRVoS application has been updated and redesigned to provide access to both manual and machine learning-based segmentation and annotation techniques, including support for crowd sourced data. Combining adjacent, similar voxels (supervoxels) provides a mechanism for speeding up segmentation both in the painting of annotation and by training a segmentation model on a small amount of annotation. The support for layers allows multiple datasets to be viewed and annotated together which, for example, enables the use of correlative data (e.g. crowd-sourced annotations or secondary imaging techniques) to guide segmentation. The ability to work with larger data on high-performance servers with GPUs has been added through a client-server architecture and the Pytorch-based image processing and segmentation server is flexible and extensible, and allows the implementation of deep learning-based segmentation modules. The client side has been built around Napari allowing integration of SuRVoS into an ecosystem for open-source image analysis while the server side has been built with cloud computing and extensibility through plugins in mind. Together these improvements to SuRVoS provide a platform for accelerating the annotation and segmentation of volumetric and correlative imaging data across modalities and scales.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Pennington, A., King, O.N.F., Tun, W.M., Ho, E.M.L., Luengo, I., Darrow, M.C., and Basham, M. SuRVoS 2: Accelerating Annotation and Segmentation for Large Volumetric Bioimage Workflows Across Modalities and Scales. Frontiers in Cell and Developmental Biology. 2022 ;10:842342. DOI: https://doi.org/10.3389/fcell.2022.842342.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.3389/fcell.2022.842342"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Beamline K11 DIAD: A new instrument for dual imaging and diffraction at Diamond Light Source
  </h2>
  <p class="mb-1">
    <span class="text-muted">2021</span>
    |
    Reinhard C, Drakopoulos M, Ahmed SI, Deyhle H, James A, Charlesworth CM, Burt M, Sutter J, Alexander S, Garland P, Yates T, Marshall R, Kemp B, Warrick E, Pueyos A, Bradnick B, Nagni M, Winter AD, Filik J, Basham M, Wadeson N, King ONF, Aslani N and Dent AJ.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Journal of Synchrotron Radiation
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1107/S1600577521009875">10.1107/S1600577521009875</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-6"
      aria-expanded="false" aria-controls="collapse-6">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1107/S1600577521009875" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-6" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>The Dual Imaging and Diffraction (DIAD) beamline at Diamond Light Source is a new dual-beam instrument for full-field imaging/tomography and powder diffraction. This instrument provides the user community with the capability to dynamically image 2D and 3D complex structures and perform phase identification and/or strain mapping using micro-diffraction. The aim is to enable in situ and in operando experiments that require spatially correlated results from both techniques, by providing measurements from the same specimen location quasi-simultaneously. Using an unusual optical layout, DIAD has two independent beams originating from one source that operate in the medium energy range (7-38 keV) and are combined at one sample position. Here, either radiography or tomography can be performed using monochromatic or pink beam, with a 1.4 mm × 1.2 mm field of view and a feature resolution of 1.2 µm. Micro-diffraction is possible with a variable beam size between 13 µm × 4 µm and 50 µm × 50 µm. One key functionality of the beamline is image-guided diffraction, a setup in which the micro-diffraction beam can be scanned over the complete area of the imaging field-of-view. This moving beam setup enables the collection of location-specific information about the phase composition and/or strains at any given position within the image/tomography field of view. The dual beam design allows fast switching between imaging and diffraction mode without the need of complicated and time-consuming mode switches. Real-time selection of areas of interest for diffraction measurements as well as the simultaneous collection of both imaging and diffraction data of (irreversible) in situ and in operando experiments are possible.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Reinhard C, Drakopoulos M, Ahmed SI, Deyhle H, James A, Charlesworth CM, Burt M, Sutter J, Alexander S, Garland P, Yates T, Marshall R, Kemp B, Warrick E, Pueyos A, Bradnick B, Nagni M, Winter AD, Filik J, Basham M, Wadeson N, King ONF, Aslani N, Dent AJ. Beamline K11 DIAD: a new instrument for dual imaging and diffraction at Diamond Light Source. J Synchrotron Radiat. 2021 Nov 1;28(Pt 6):1985-1995. doi: https://doi.org/10.1107/S1600577521009875. Epub 2021 Oct 22.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1107/S1600577521009875"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    A massively multi-scale approach to characterising tissue architecture by synchrotron micro-CT applied to the human placenta
  </h2>
  <p class="mb-1">
    <span class="text-muted">2021</span>
    |
    Tun WM, Poologasundarampillai G, Bischof H, Nye G, King ONF, Basham M, Tokudome Y, Lewis RM, Johnstone ED, Brownbill P, Darrow M and Chernyavsky IL.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Journal of the Royal Society Interface
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1098/rsif.2021.0140">10.1098/rsif.2021.0140</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-7"
      aria-expanded="false" aria-controls="collapse-7">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1098/rsif.2021.0140" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-7" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Multi-scale structural assessment of biological soft tissue is challenging but essential to gain insight into structure–function relationships of tissue/organ. Using the human placenta as an example, this study brings together sophisticated sample preparation protocols, advanced imaging and robust, validated machine-learning segmentation techniques to provide the first massively multi-scale and multi-domain information that enables detailed morphological and functional analyses of both maternal and fetal placental domains. Finally, we quantify the scale-dependent error in morphological metrics of heterogeneous placental tissue, estimating the minimal tissue scale needed in extracting meaningful biological data. The developed protocol is beneficial for high-throughput investigation of structure–function relationships in both normal and diseased placentas, allowing us to optimize therapeutic approaches for pathological pregnancies. In addition, the methodology presented is applicable in the characterization of tissue architecture and physiological behaviours of other complex organs with similarity to the placenta, where an exchange barrier possesses circulating vascular and avascular fluid spaces.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Tun WM, Poologasundarampillai G, Bischof H, Nye G, King ONF, Basham M, Tokudome Y, Lewis RM, Johnstone ED, Brownbill P, Darrow M, Chernyavsky IL. A massively multi-scale approach to characterizing tissue architecture by synchrotron micro-CT applied to the human placenta. J R Soc Interface. 2021 Jun;18(179):20210140. doi: https://doi.org/10.1098/rsif.2021.0140. Epub 2021 Jun 2.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1098/rsif.2021.0140"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Non-competitive cyclic peptides for targeting enzyme-substrate complexes
  </h2>
  <p class="mb-1">
    <span class="text-muted">2018</span>
    |
    McAllister TE, Yeh TL, Abboud MI, Leung IKH, Hookway ES, King ONF, Bhushan B, Williams ST, Hopkinson RJ, Münzel M, Loik ND, Chowdhury R, Oppermann U, Claridge TDW, Goto Y, Suga H, Schofield CJ and Kawamura A.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Chemical Science
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1039/c8sc00286j">10.1039/c8sc00286j</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-8"
      aria-expanded="false" aria-controls="collapse-8">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1039/c8sc00286j" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-8" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Affinity reagents are of central importance for selectively identifying proteins and investigating their interactions. We report on the development and use of cyclic peptides, identified by mRNA display-based RaPID methodology, that are selective for, and tight binders of, the human hypoxia inducible factor prolyl hydroxylases (PHDs) – enzymes crucial in hypoxia sensing. Biophysical analyses reveal the cyclic peptides to bind in a distinct site, away from the enzyme active site pocket, enabling conservation of substrate binding and catalysis. A biotinylated cyclic peptide captures not only the PHDs, but also their primary substrate hypoxia inducible factor HIF1-α. Our work highlights the potential for tight, non-active site binding cyclic peptides to act as promising affinity reagents for studying protein–protein interactions.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">McAllister TE, Yeh TL, Abboud MI, Leung IKH, Hookway ES, King ONF, Bhushan B, Williams ST, Hopkinson RJ, Münzel M, Loik ND, Chowdhury R, Oppermann U, Claridge TDW, Goto Y, Suga H, Schofield CJ, Kawamura A. Non-competitive cyclic peptides for targeting enzyme-substrate complexes. Chem Sci. 2018 Apr 23;9(20):4569-4578. doi: https://doi.org/10.1039/c8sc00286j.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1039/c8sc00286j"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Highly selective inhibition of histone demethylases by de novo macrocyclic peptides
  </h2>
  <p class="mb-1">
    <span class="text-muted">2017</span>
    |
    Kawamura A, Münzel M, Kojima T, Yapp C, Bhushan B, Goto Y, Tumber A, Katoh T, King ON, Passioura T, Walport LJ, Hatch SB, Madden S, Müller S, Brennan PE, Chowdhury R, Hopkinson RJ, Suga H and Schofield CJ.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Nature Communications
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1038/ncomms14773">10.1038/ncomms14773</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-9"
      aria-expanded="false" aria-controls="collapse-9">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1038/ncomms14773" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-9" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>The JmjC histone demethylases (KDMs) are linked to tumour cell proliferation and are current cancer targets; however, very few highly selective inhibitors for these are available. Here we report cyclic peptide inhibitors of the KDM4A-C with selectivity over other KDMs/2OG oxygenases, including closely related KDM4D/E isoforms. Crystal structures and biochemical analyses of one of the inhibitors (CP2) with KDM4A reveals that CP2 binds differently to, but competes with, histone substrates in the active site. Substitution of the active site binding arginine of CP2 to N-ɛ-trimethyl-lysine or methylated arginine results in cyclic peptide substrates, indicating that KDM4s may act on non-histone substrates. Targeted modifications to CP2 based on crystallographic and mass spectrometry analyses results in variants with greater proteolytic robustness. Peptide dosing in cells manifests KDM4A target stabilization. Although further development is required to optimize cellular activity, the results reveal the feasibility of highly selective non-metal chelating, substrate-competitive inhibitors of the JmjC KDMs.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Kawamura A, Münzel M, Kojima T, Yapp C, Bhushan B, Goto Y, Tumber A, Katoh T, King ON, Passioura T, Walport LJ, Hatch SB, Madden S, Müller S, Brennan PE, Chowdhury R, Hopkinson RJ, Suga H, Schofield CJ. Highly selective inhibition of histone demethylases by de novo macrocyclic peptides. Nat Commun 8, 14773 (2017). https://doi.org/10.1038/ncomms14773</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1038/ncomms14773"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Pan-histone demethylase inhibitors simultaneously targeting Jumonji C and lysine-specific demethylases display high anticancer activities
  </h2>
  <p class="mb-1">
    <span class="text-muted">2014</span>
    |
    Rotili D, Tomassi S, Conte M, Benedetti R, Tortorici M, Ciossani G, Valente S, Marrocco B, Labella D, Novellino E, Mattevi A, Altucci L, Tumber A, Yapp C, King ON, Hopkinson RJ, Kawamura A, Schofield CJ and Mai A.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Journal of Medicinal Chemistry
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1021/jm4012802">10.1021/jm4012802</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-10"
      aria-expanded="false" aria-controls="collapse-10">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1021/jm4012802" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-10" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>In prostate cancer, two different types of histone lysine demethylases (KDM), LSD1/KDM1 and JMJD2/KDM4, are coexpressed and colocalize with the androgen receptor. We designed and synthesized hybrid LSD1/JmjC or "pan-KDM" inhibitors 1-6 by coupling the skeleton of tranylcypromine 7, a known LSD1 inhibitor, with 4-carboxy-4'-carbomethoxy-2,2'-bipyridine 8 or 5-carboxy-8-hydroxyquinoline 9, two 2-oxoglutarate competitive templates developed for JmjC inhibition. Hybrid compounds 1-6 are able to simultaneously target both KDM families and have been validated as potential antitumor agents in cells. Among them, 2 and 3 increase H3K4 and H3K9 methylation levels in cells and cause growth arrest and substantial apoptosis in LNCaP prostate and HCT116 colon cancer cells. When tested in noncancer mesenchymal progenitor (MePR) cells, 2 and 3 induced little and no apoptosis, respectively, thus showing cancer-selective inhibiting action.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Rotili D, Tomassi S, Conte M, Benedetti R, Tortorici M, Ciossani G, Valente S, Marrocco B, Labella D, Novellino E, Mattevi A, Altucci L, Tumber A, Yapp C, King ON, Hopkinson RJ, Kawamura A, Schofield CJ, Mai A. Pan-histone demethylase inhibitors simultaneously targeting Jumonji C and lysine-specific demethylases display high anticancer activities. J Med Chem. 2014 Jan 9;57(1):42-55. doi:  https://doi.org/10.1021/jm4012802.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1021/jm4012802"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation
  </h2>
  <p class="mb-1">
    <span class="text-muted">2013</span>
    |
    Hopkinson RJ,  Tumber A, Yapp C, Chowdhury R, Aik WS, Che KH, Li XS, Kristensen JBL, King ONF, Chan MC, Yeoh KK, Choi H, Walport LJ, Thinnes CC, Bush JT, Lejeune C,  Rydzik AM, Rose NR,  Bagg EA, McDonough MA, Krojer, TJ, Yue WW, Ng SS, Olsen L, Brennan PE, Oppermann U, Muller S, Klose RJ, Ratcliffe PJ, Schofield CJ and Kawamura A.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Chemical Science
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1039/C3SC51122G">10.1039/C3SC51122G</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-11"
      aria-expanded="false" aria-controls="collapse-11">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1039/C3SC51122G" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-11" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>2-Oxoglutarate and iron dependent oxygenases are therapeutic targets for human diseases. Using a representative 2OG oxygenase panel, we compare the inhibitory activities of 5-carboxy-8-hydroxyquinoline (IOX1) and 4-carboxy-8-hydroxyquinoline (4C8HQ) with that of two other commonly used 2OG oxygenase inhibitors, N-oxalylglycine (NOG) and 2,4-pyridinedicarboxylic acid (2,4-PDCA). The results reveal that IOX1 has a broad spectrum of activity, as demonstrated by the inhibition of transcription factor hydroxylases, representatives of all 2OG dependent histone demethylase subfamilies, nucleic acid demethylases and γ-butyrobetaine hydroxylase. Cellular assays show that, unlike NOG and 2,4-PDCA, IOX1 is active against both cytosolic and nuclear 2OG oxygenases without ester derivatisation. Unexpectedly, crystallographic studies on these oxygenases demonstrate that IOX1, but not 4C8HQ, can cause translocation of the active site metal, revealing a rare example of protein ligand-induced metal movement.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Hopkinson RJ, Tumber A, Yapp C, Chowdhury R, Aik W, Che KH, Li XS, Kristensen JBL, King ONF, Chan MC, Yeoh KK, Choi H, Walport LJ, Thinnes CC, Bush JT, Lejeune C, Rydzik AM, Rose NR, Bagg EA, McDonough MA, Krojer T, Yue WW, Ng SS, Olsen L, Brennan PE, Oppermann U, Muller-Knapp S, Klose RJ, Ratcliffe PJ, Schofield CJ, Kawamura A. 5-Carboxy-8-hydroxyquinoline is a Broad Spectrum 2-Oxoglutarate Oxygenase Inhibitor which Causes Iron Translocation. Chem Sci. 2013 Aug 1;4(8):3110-3117. doi: https://doi.org/10.1039/C3SC51122G.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1039/C3SC51122G"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Discovery of ML324, a JMJD2 demethylase inhibitor with demonstrated antiviral activity
  </h2>
  <p class="mb-1">
    <span class="text-muted">2012</span>
    |
    Rai G, Kawamura A, Tumber A, Liang Y, Vogel JL, Arbuckle JH, Rose NR, Dexheimer TS, Foley TL, King ON, Quinn A, Mott BT, Schofield CJ, Oppermann U, Jadhav A, Simeonov A, Kristie TM and Maloney DJ.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Probe Reports from the NIH Molecular Libraries Program [Internet].
    </b>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-12"
      aria-expanded="false" aria-controls="collapse-12">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://www.ncbi.nlm.nih.gov/books/NBK169450/" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-12" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>A critical and dynamic epigenetic post-translational modification involves N(ε)-methylation of histone lysine residues by histone methyltransferases. This process was originally thought to be an irreversible epigenetic mark, yet two representative classes of histone lysine demethylases which reverse this process are LSD1/2 and the Jumonji domain containing proteins (JMJD) have emerged. Despite an increased interest in these enzymes as a result of their suspected role in a variety of diseases (e.g. cancer and virus infection), a dearth of potent and cell-permeable inhibitors of the JMJD2 enzymes remain. As such, we sought to discover novel small molecule inhibitors of the JMJD2 family of histone demethylases via a quantitative high throughput screen and subsequent medicinal chemistry optimization campaign. Herein, we describe the discovery and optimization of N-(3-(dimethyamino)propyl-4-(8-hydroxyquinolin-6-yl)benzamide, ML324, a probe molecule that displays submicromolar inhibitory activity toward JMJD2E (in vitro) and possesses excellent in vitro ADME properties. In contrast to previously reported inhibitors of the JMJD proteins, ML324 displays excellent cell permeability providing an opportunity for more extensive cell-based studies of JMJD2 enzymes to be undertaken. In addition, ML324 demonstrates potent anti-viral activity against both herpes simplex virus (HSV) and human cytomegalovirus (hCMV) infection via inhibition viral IE gene expression. ML324 suppresses the formation of HSV plaques, even at high MOI, and blocks HSV-1 reactivation in a mouse ganglia explant model of latently infected mice. The studies described herein provide the basis for the use of JMJD2 inhibitors in proof-of-concept animal models for treatment of herpes virus infections and recurrence.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Rai G, Kawamura A, Tumber A, Liang Y, Vogel JL, Arbuckle JH, Rose NR, Dexheimer TS, Foley TL, King ON, Quinn A, Mott BT, Schofield CJ, Oppermann U, Jadhav A, Simeonov A, Kristie TM, Maloney DJ. Discovery of ML324, a JMJD2 demethylase inhibitor with demonstrated antiviral activity. 2012 Dec 17 [updated 2013 Sep 16]. In: Probe Reports from the NIH Molecular Libraries Program [Internet]. Bethesda (MD): National Center for Biotechnology Information (US); 2010–.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Plant growth regulator daminozide is a selective inhibitor of human KDM2/7 histone demethylases
  </h2>
  <p class="mb-1">
    <span class="text-muted">2012</span>
    |
    Rose NR, Woon EC, Tumber A, Walport LJ, Chowdhury R, Li XS, King ON, Lejeune C, Ng SS, Krojer T, Chan MC, Rydzik AM, Hopkinson RJ, Che KH, Daniel M, Strain-Damerell C, Gileadi C, Kochan G, Leung IK, Dunford J, Yeoh KK, Ratcliffe PJ, Burgess-Brown N, von Delft F, Muller S, Marsden B, Brennan PE, McDonough MA, Oppermann U, Klose RJ, Schofield CJ and Kawamura A.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Journal of Medicinal Chemistry
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1021/jm300677j">10.1021/jm300677j</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-13"
      aria-expanded="false" aria-controls="collapse-13">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1021/jm300677j" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-13" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>The JmjC oxygenases catalyse the N-demethylation of Nε-methyl lysine residues in histones and are current therapeutic targets. A set of 2-oxoglutarate analogues were screened using a unified assay platform for JmjC demethylases and related oxygenases. Results led to the finding that daminozide (N-(dimethylamino)succinamic acid, 160 Da), a plant growth regulator, selectively inhibits the KDM2/7 JmjC subfamily. Kinetic and crystallographic studies reveal daminozide chelates the active site metal via its hydrazide carbonyl and dimethylamino groups.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Rose NR, Woon EC, Tumber A, Walport LJ, Chowdhury R, Li XS, King ON, Lejeune C, Ng SS, Krojer T, Chan MC, Rydzik AM, Hopkinson RJ, Che KH, Daniel M, Strain-Damerell C, Gileadi C, Kochan G, Leung IK, Dunford J, Yeoh KK, Ratcliffe PJ, Burgess-Brown N, von Delft F, Muller S, Marsden B, Brennan PE, McDonough MA, Oppermann U, Klose RJ, Schofield CJ, Kawamura A. Plant growth regulator daminozide is a selective inhibitor of human KDM2/7 histone demethylases. J Med Chem. 2012 Jul 26;55(14):6639-43. doi: https://doi.org/10.1021/jm300677j. Epub 2012 Jul 11.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1021/jm300677j"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    The Cryptosporidium parvum kinome
  </h2>
  <p class="mb-1">
    <span class="text-muted">2011</span>
    |
    Artz JD, Wernimont AK, Allali-Hassani A, Zhao Y, Amani M, Lin YH, Senisterra G, Wasney GA, Fedorov O, King O, Roos A, Lunin VV, Qiu W, Finerty P Jr, Hutchinson A, Chau I, von Delft F, MacKenzie F, Lew J, Kozieradzki I, Vedadi M, Schapira M, Zhang C, Shokat K, Heightman T and Hui R.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    BMC Genomics
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1186/1471-2164-12-478">10.1186/1471-2164-12-478</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-14"
      aria-expanded="false" aria-controls="collapse-14">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1186/1471-2164-12-478" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-14" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Background:
Hundreds of millions of people are infected with cryptosporidiosis annually, with immunocompromised individuals suffering debilitating symptoms and children in socioeconomically challenged regions at risk of repeated infections. There is currently no effective drug available. In order to facilitate the pursuit of anti-cryptosporidiosis targets and compounds, our study spans the classification of the Cryptosporidium parvum kinome and the structural and biochemical characterization of representatives from the CDPK family and a MAP kinase.
Results:
The C. parvum kinome comprises over 70 members, some of which may be promising drug targets. These C. parvum protein kinases include members in the AGC, Atypical, CaMK, CK1, CMGC, and TKL groups; however, almost 35% could only be classified as OPK (other protein kinases). In addition, about 25% of the kinases identified did not have any known orthologues outside of Cryptosporidium spp. Comparison of specific kinases with their Plasmodium falciparum and Toxoplasma gondii orthologues revealed some distinct characteristics within the C. parvum kinome, including potential targets and opportunities for drug design. Structural and biochemical analysis of 4 representatives of the CaMK group and a MAP kinase confirms features that may be exploited in inhibitor design. Indeed, screening Cp CDPK1 against a library of kinase inhibitors yielded a set of the pyrazolopyrimidine derivatives (PP1-derivatives) with IC50 values of < 10 nM. The binding of a PP1-derivative is further described by an inhibitor-bound crystal structure of Cp CDPK1. In addition, structural analysis of Cp CDPK4 identified an unprecedented Zn-finger within the CDPK kinase domain that may have implications for its regulation.
Conclusions:
Identification and comparison of the C. parvum protein kinases against other parasitic kinases shows how orthologue- and family-based research can be used to facilitate characterization of promising drug targets and the search for new drugs.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Artz JD, Wernimont AK, Allali-Hassani A, Zhao Y, Amani M, Lin YH, Senisterra G, Wasney GA, Fedorov O, King O, Roos A, Lunin VV, Qiu W, Finerty P Jr, Hutchinson A, Chau I, von Delft F, MacKenzie F, Lew J, Kozieradzki I, Vedadi M, Schapira M, Zhang C, Shokat K, Heightman T, Hui R. The Cryptosporidium parvum kinome. BMC Genomics. 2011 Sep 30;12:478. doi: https://doi.org/10.1186/1471-2164-12-478.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1186/1471-2164-12-478"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Inhibition of 2-oxoglutarate dependent oxygenases
  </h2>
  <p class="mb-1">
    <span class="text-muted">2011</span>
    |
    Rose NR, McDonough MA, King ON, Kawamura A and Schofield CJ.
  </p>
  <p class="text-muted fs-6">
    <b>
    Chemical Society Reviews
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1039/c0cs00203h">10.1039/c0cs00203h</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-15"
      aria-expanded="false" aria-controls="collapse-15">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="/pdf/Chem_Soc_Rev.pdf" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-15" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidation to the conversion of 2OG to succinate and carbon dioxide. In humans their roles include collagen biosynthesis, fatty acid metabolism, DNA repair, RNA and chromatin modifications, and hypoxic sensing. Commercial applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clinically used pharmaceutical compound for cardioprotection. Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anaemia, inflammation and cancer. In this critical review, we describe studies on the inhibition of 2OG oxygenases, focusing on small molecules, and discuss the potential of 2OG oxygenases as therapeutic targets (295 references).</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Rose NR, McDonough MA, King ON, Kawamura A, Schofield CJ. Inhibition of 2-oxoglutarate dependent oxygenases. Chem Soc Rev. 2011 Aug;40(8):4364-97. doi: https://doi.org/10.1039/c0cs00203h. Epub 2011 Mar 10.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1039/c0cs00203h"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases
  </h2>
  <p class="mb-1">
    <span class="text-muted">2011</span>
    |
    Chowdhury R, Yeoh KK, Tian YM, Hillringhaus L, Bagg EA, Rose NR, Leung IK, Li XS, Woon EC, Yang M, McDonough MA, King ON, Clifton IJ, Klose RJ, Claridge TD, Ratcliffe PJ, Schofield CJ and Kawamura A.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    EMBO Reports
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1038/embor.2011.43">10.1038/embor.2011.43</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-16"
      aria-expanded="false" aria-controls="collapse-16">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1038/embor.2011.43" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-16" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Mutations in isocitrate dehydrogenases (IDHs) have a gain-of-function effect leading to R(−)-2-hydroxyglutarate (R-2HG) accumulation. By using biochemical, structural and cellular assays, we show that either or both R- and S-2HG inhibit 2-oxoglutarate (2OG)-dependent oxygenases with varying potencies. Half-maximal inhibitory concentration (IC50) values for the R-form of 2HG varied from approximately 25 μM for the histone Nε-lysine demethylase JMJD2A to more than 5 mM for the hypoxia-inducible factor (HIF) prolyl hydroxylase. The results indicate that candidate oncogenic pathways in IDH-associated malignancy should include those that are regulated by other 2OG oxygenases than HIF hydroxylases, in particular those involving the regulation of histone methylation.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Chowdhury R, Yeoh KK, Tian YM, Hillringhaus L, Bagg EA, Rose NR, Leung IK, Li XS, Woon EC, Yang M, McDonough MA, King ON, Clifton IJ, Klose RJ, Claridge TD, Ratcliffe PJ, Schofield CJ, Kawamura A. The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases. EMBO Rep. 2011 May;12(5):463-9. doi: https://doi.org/10.1038/embor.2011.43. Epub 2011 Apr 1.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1038/embor.2011.43"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Inhibition of histone demethylases by 4-carboxy-2,2'-bipyridyl compounds
  </h2>
  <p class="mb-1">
    <span class="text-muted">2011</span>
    |
    Chang KH*, King ON*, Tumber A*, Woon EC, Heightman TD, McDonough MA, Schofield CJ and Rose NR.
  </p>
  <p class="text-muted fs-6">
    <b>
    ChemMedChem
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1002/cmdc.201100026">10.1002/cmdc.201100026</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-17"
      aria-expanded="false" aria-controls="collapse-17">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="/pdf/emss-66234.pdf" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-17" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Exploiting epigenetics: 2-Oxoglutarate (2OG)-dependent histone lysine demethylases, such as JMJD2E, are potential therapeutic targets in a range of diseases. Through structure–activity relationship studies and analyses, we identified a potent 4-carboxy-2,2′-bipyridyl compound, which inhibits JMJD2E with an IC50 value of 110 nM, representing a 66-fold improvement over the lead compound. These bipyridyl derivatives bind in the 2-oxoglutarate binding site.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Chang KH, King ONF, Tumber A, Woon ECY, Heightman TD, McDonough MA, Schofield CJ, Rose NR. Inhibition of histone demethylases by 4-carboxy-2,2'-bipyridyl compounds. ChemMedChem. 2011 May 2;6(5):759-764. doi:  https://doi.org/10.1002/cmdc.201100026. Epub 2011 Mar 15.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1002/cmdc.201100026"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    A prominent β-hairpin structure in the winged-helix domain of RECQ1 is required for DNA unwinding and oligomer formation
  </h2>
  <p class="mb-1">
    <span class="text-muted">2011</span>
    |
    Lucic B, Zhang Y, King O, Mendoza-Maldonado R, Berti M, Niesen FH, Burgess-Brown NA, Pike AC, Cooper CD, Gileadi O and Vindigni A.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Nucleic Acids Research
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1093/nar/gkq1031">10.1093/nar/gkq1031</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-18"
      aria-expanded="false" aria-controls="collapse-18">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1093/nar/gkq1031" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-18" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>RecQ helicases have attracted considerable interest in recent years due to their role in the suppression of genome instability and human diseases. These atypical helicases exert their function by resolving a number of highly specific DNA structures. The crystal structure of a truncated catalytic core of the human RECQ1 helicase (RECQ1 49–616 ) shows a prominent β-hairpin, with an aromatic residue (Y564) at the tip, located in the C-terminal winged-helix domain. Here, we show that the β-hairpin is required for the DNA unwinding and Holliday junction (HJ) resolution activity of full-length RECQ1, confirming that it represents an important determinant for the distinct substrate specificity of the five human RecQ helicases. In addition, we found that the β-hairpin is required for dimer formation in RECQ1 49–616 and tetramer formation in full-length RECQ1. We confirmed the presence of stable RECQ1 49–616 dimers in solution and demonstrated that dimer formation favours DNA unwinding; even though RECQ1 monomers are still active. Tetramers are instead necessary for more specialized activities such as HJ resolution and strand annealing. Interestingly, two independent protein–protein contacts are required for tetramer formation, one involves the β-hairpin and the other the N-terminus of RECQ1, suggesting a non-hierarchical mechanism of tetramer assembly.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Lucic B, Zhang Y, King O, Mendoza-Maldonado R, Berti M, Niesen FH, Burgess-Brown NA, Pike AC, Cooper CD, Gileadi O, Vindigni A. A prominent β-hairpin structure in the winged-helix domain of RECQ1 is required for DNA unwinding and oligomer formation. Nucleic Acids Res. 2011 Mar;39(5):1703-17. doi: https://doi.org/10.1093/nar/gkq1031.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1093/nar/gkq1031"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Specific CLK inhibitors from a novel chemotype for regulation of alternative splicing
  </h2>
  <p class="mb-1">
    <span class="text-muted">2011</span>
    |
    Fedorov O, Huber K, Eisenreich A, Filippakopoulos P, King O, Bullock AN, Szklarczyk D, Jensen LJ, Fabbro D, Trappe J, Rauch U, Bracher F and Knapp S.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Chemistry & Biology
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1016/j.chembiol.2010.11.009">10.1016/j.chembiol.2010.11.009</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-19"
      aria-expanded="false" aria-controls="collapse-19">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1016/j.chembiol.2010.11.009" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-19" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>There is a growing recognition of the importance of protein kinases in the control of alternative splicing. To define the underlying regulatory mechanisms, highly selective inhibitors are needed. Here, we report the discovery and characterization of the dichloroindolyl enaminonitrile KH-CB19, a potent and highly specific inhibitor of the CDC2-like kinase isoforms 1 and 4 (CLK1/CLK4). Co-crystal structures of KH-CB19 with CLK1 and CLK3 revealed a non-ATP mimetic binding mode, conformational changes in helix αC and the phosphate binding loop and halogen bonding to the kinase hinge region. KH-CB19 effectively suppressed phosphorylation of SR (serine/arginine) proteins in cells, consistent with its expected mechanism of action. Chemical inhibition of CLK1/CLK4 generated a unique pattern of splicing factor dephosphorylation and had at low nM concentration a profound effect on splicing of the two tissue factor isoforms flTF (full-length TF) and asHTF (alternatively spliced human TF).</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Fedorov O, Huber K, Eisenreich A, Filippakopoulos P, King O, Bullock AN, Szklarczyk D, Jensen LJ, Fabbro D, Trappe J, Rauch U, Bracher F, Knapp S. Specific CLK inhibitors from a novel chemotype for regulation of alternative splicing. Chem Biol. 2011 Jan 28;18(1):67-76. doi: https://doi.org/10.1016/j.chembiol.2010.11.009.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1016/j.chembiol.2010.11.009"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors
  </h2>
  <p class="mb-1">
    <span class="text-muted">2010</span>
    |
    King ON*, Li XS*, Sakurai M*, Kawamura A, Rose NR, Ng SS, Quinn AM, Rai G, Mott BT, Beswick P, Klose RJ, Oppermann U, Jadhav A, Heightman TD, Maloney DJ, Schofield CJ and Simeonov A.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    PLoS One
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1371/journal.pone.0015535">10.1371/journal.pone.0015535</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-20"
      aria-expanded="false" aria-controls="collapse-20">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1371/journal.pone.0015535" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-20" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Background: Small molecule modulators of epigenetic processes are currently sought as basic probes for biochemical mechanisms, and as starting points for development of therapeutic agents. N(ε)-Methylation of lysine residues on histone tails is one of a number of post-translational modifications that together enable transcriptional regulation. Histone lysine demethylases antagonize the action of histone methyltransferases in a site- and methylation state-specific manner. N(ε)-Methyllysine demethylases that use 2-oxoglutarate as co-factor are associated with diverse human diseases, including cancer, inflammation and X-linked mental retardation; they are proposed as targets for the therapeutic modulation of transcription. There are few reports on the identification of templates that are amenable to development as potent inhibitors in vivo and large diverse collections have yet to be exploited for the discovery of demethylase inhibitors. Principal findings: High-throughput screening of a ∼236,000-member collection of diverse molecules arrayed as dilution series was used to identify inhibitors of the JMJD2 (KDM4) family of 2-oxoglutarate-dependent histone demethylases. Initial screening hits were prioritized by a combination of cheminformatics, counterscreening using a coupled assay enzyme, and orthogonal confirmatory detection of inhibition by mass spectrometric assays. Follow-up studies were carried out on one of the series identified, 8-hydroxyquinolines, which were shown by crystallographic analyses to inhibit by binding to the active site Fe(II) and to modulate demethylation at the H3K9 locus in a cell-based assay. Conclusions: These studies demonstrate that diverse compound screening can yield novel inhibitors of 2OG dependent histone demethylases and provide starting points for the development of potent and selective agents to interrogate epigenetic regulation.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">King ON, Li XS, Sakurai M, Kawamura A, Rose NR, Ng SS, Quinn AM, Rai G, Mott BT, Beswick P, Klose RJ, Oppermann U, Jadhav A, Heightman TD, Maloney DJ, Schofield CJ, Simeonov A. Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors. PLoS One. 2010 Nov 23;5(11):e15535. doi:  https://doi.org/10.1371/journal.pone.0015535.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0015535"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Development of homogeneous luminescence assays for histone demethylase catalysis and binding.
  </h2>
  <p class="mb-1">
    <span class="text-muted">2010</span>
    |
    Kawamura A, Tumber A, Rose NR, King ON, Daniel M, Oppermann U, Heightman TD and Schofield CJ.
  </p>
  <p class="text-muted fs-6">
    <b>
    Analytical Biochemistry
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1016/j.ab.2010.04.030">10.1016/j.ab.2010.04.030</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-21"
      aria-expanded="false" aria-controls="collapse-21">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="/pdf/emss-66189.pdf" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-21" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Covalent modifications to histones play important roles in chromatin dynamics and the regulation of gene expression. The JumonjiC (JmjC)-containing histone demethylases (HDMs) catalyze the demethylation of methylated lysine residues on histone tails. Here we report the development of homogeneous luminescence-based assay methods for measuring the catalytic activity and the binding affinities of peptides to HDMs. The assays use amplified luminescent proximity homogeneous assay (ALPHA) technology, are sensitive and robust, and can be used for small molecule inhibitor screening of HDMs. We have profiled known inhibitors of JMJD2E and demonstrate a correlation between the inhibitor potencies determined by the ALPHA and other types of assays. Although this study focuses on the JMJD2E isoform, the catalytic turnover and binding assays described here can be used in studies on other HDMs. The assays should be useful for the development of small molecule inhibitors selective for HDM isoforms.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Kawamura A, Tumber A, Rose NR, King ON, Daniel M, Oppermann U, Heightman TD, Schofield C. Development of homogeneous luminescence assays for histone demethylase catalysis and binding. Anal Biochem. 2010 Sep 1;404(1):86-93. doi: https://doi.org/10.1016/j.ab.2010.04.030.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ab.2010.04.030"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Selective inhibitors of the JMJD2 histone demethylases: Combined nondenaturing mass spectrometric screening and crystallographic approaches
  </h2>
  <p class="mb-1">
    <span class="text-muted">2010</span>
    |
    Rose NR, Woon EC, Kingham GL, King ON, Mecinović J, Clifton IJ, Ng SS, Talib-Hardy J, Oppermann U, McDonough MA, and Schofield CJ.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Journal of Medicinal Chemistry
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1021/jm901680b">10.1021/jm901680b</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-22"
      aria-expanded="false" aria-controls="collapse-22">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1021/jm901680b" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-22" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Ferrous ion and 2-oxoglutarate (2OG) oxygenases catalyze the demethylation of Nε-methylated lysine residues in histones. Here we report studies on the inhibition of the JMJD2 subfamily of histone demethylases, employing binding analyses by nondenaturing mass spectrometry (MS), dynamic combinatorial chemistry coupled to MS, turnover assays, and crystallography. The results of initial binding and inhibition assays directed the production and analysis of a set of N-oxalyl-d-tyrosine derivatives to explore the extent of a subpocket at the JMJD2 active site. Some of the inhibitors were shown to be selective for JMJD2 over the hypoxia-inducible factor prolyl hydroxylase PHD2. A crystal structure of JMJD2A in complex with one of the potent inhibitors was obtained; modeling other inhibitors based on this structure predicts interactions that enable improved inhibition for some compounds.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Nathan R. Rose, Esther C. Y. Woon, Guy L. Kingham, Oliver N. F. King, Jasmin Mecinović, Ian J. Clifton, Stanley S. Ng, Jobina Talib-Hardy, Udo Oppermann, Michael A. McDonough, and Christopher J. Schofield. Selective inhibitors of the JMJD2 histone demethylases: Combined nondenaturing mass spectrometric screening and crystallographic approaches. Journal of Medicinal Chemistry 2010 53 (4), 1810-1818. https://doi.org/10.1021/jm901680b</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1021/jm901680b"></span>
  </div>
</div>
<div class="mb-5">
  <h2 class="fs-4 mb-1">
    Large Scale Structural Analysis of the Classical Human Protein Tyrosine Phosphatome
  </h2>
  <p class="mb-1">
    <span class="text-muted">2009</span>
    |
    Barr AJ, Ugochukwu E, Lee WH, King ON, Filippakopoulos P, Alfano I, Savitsky P, Burgess-Brown NA, Müller S and Knapp S.
  </p>
  <p class="text-muted fs-6">
      <span class="badge bg-success fw-normal me-1">
        <i class="ai ai-open-access me-1" aria-hidden="true"></i>
        open-access
      </span>
    <b>
    Cell
    </b>
      doi:<a target="_blank" href="https://doi.org/10.1016/j.cell.2008.11.038">10.1016/j.cell.2008.11.038</a>
  </p>
  <button class="btn btn-secondary btn-sm me-1 mb-2" type="button"
      data-bs-toggle="collapse" data-bs-target="#collapse-23"
      aria-expanded="false" aria-controls="collapse-23">
    More info <i class="fa fa-chevron-circle-down ms-1" aria-hidden="true"></i>
  </button>
  <a class="btn btn-sm btn-primary me-1 mb-2" href="https://doi.org/10.1016/j.cell.2008.11.038" target="_blank">
    <i class="fa fa-file-pdf me-1" aria-hidden="true"></i>
    PDF
  </a>
  <div id="collapse-23" class="collapse paper-info mt-2 overflow-hidden">
    <h3 class="fs-4">Abstract</h3>
    <p>Protein tyrosine phosphatases (PTPs) play a critical role in regulating cellular functions by selectively dephosphorylating their substrates. Here we present 22 human PTP crystal structures that, together with prior structural knowledge, enable a comprehensive analysis of the classical PTP family. Despite their largely conserved fold, surface properties of PTPs are strikingly diverse. A potential secondary substrate-binding pocket is frequently found in phosphatases, and this has implications for both substrate recognition and development of selective inhibitors. Structural comparison identified four diverse catalytic loop (WPD) conformations and suggested a mechanism for loop closure. Enzymatic assays revealed vast differences in PTP catalytic activity and identified PTPD1, PTPD2, and HDPTP as catalytically inert protein phosphatases. We propose a “head-to-toe” dimerization model for RPTPγ/ζ that is distinct from the “inhibitory wedge” model and that provides a molecular basis for inhibitory regulation. This phosphatome resource gives an expanded insight into intrafamily PTP diversity, catalytic activity, substrate recognition, and autoregulatory self-association.</p>
    <h3 class="fs-4">Cite as</h3>
    <blockquote class="mb-4">Barr AJ, Ugochukwu E, Lee WH, King ON, Filippakopoulos P, Alfano I, Savitsky P, Burgess-Brown NA, Müller S, Knapp S. Large-scale structural analysis of the classical human protein tyrosine phosphatome. Cell. 2009 Jan 23;136(2),352-63. https://doi.org/10.1016/j.cell.2008.11.038.</blockquote>
    <h3 class="fs-4 mb-4">Citations</h3>
    <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2008.11.038"></span>
  </div>
</div>
</div>
      </section>
  </main>

  <footer>
    <div class="container">
      <nav class="footer-nav mb-4" aria-labelledby="secondary-navigation">
        <ul class="nav">
            <li class="nav-item">
              
  
  
  
  <a class="nav-link"  href="/">Home</a>
            </li>
            <li class="nav-item">
              
  
  
  
  <a class="nav-link"  href="/about/">About</a>
            </li>
            <li class="nav-item">
              
  
  
  
    
    
  
  <a class="nav-link active" aria-current="page" href="/publications/">Publications</a>
            </li>
        </ul>
      </nav>
      <ul class="list-inline mb-5">
        <li class="list-inline-item">
          <a aria-label="Twitter" href="https://twitter.com/_OllyK" target="_blank"><i class="fab fa-twitter fa-lg me-2" title="Twitter" aria-hidden="true"></i></a>
        </li>
        <li class="list-inline-item">
          <a aria-label="GitHub" href="https://github.com/OllyK" target="_blank"><i class="fab fa-github fa-lg me-2" title="GitHub" aria-hidden="true"></i></a>
        </li>
        <li class="list-inline-item">
          <a aria-label="Flickr" href="https://www.flickr.com/photos/mixedphase" target="_blank"><i class="fab fa-flickr fa-lg me-2" title="Flickr" aria-hidden="true"></i></a>
        </li>
        <li class="list-inline-item">
          <a aria-label="Instagram" href="https://instagram.com/olly.king" target="_blank"><i class="fab fa-instagram fa-lg me-2" title="Instagram" aria-hidden="true"></i></a>
        </li>
        <li class="list-inline-item">
          <a aria-label="LinkedIn" href="https://uk.linkedin.com/in/oliverking2" target="_blank"><i class="fab fa-linkedin fa-lg me-2" title="LinkedIn" aria-hidden="true"></i></a>
        </li>
        <li class="list-inline-item">
          <a aria-label="ORCID" href="https://orcid.org/0000-0002-6152-7207" target="_blank"><i class="ai ai-orcid fa-lg me-2" title="ORCID" aria-hidden="true"></i></a>
        </li>
      </ul>
      <p>
        Copyright &copy; 2024 Oliver King.
          Except where otherwise noted, all content is licensed
<a href="https://creativecommons.org/licenses/by/4.0/">CC-BY</a>.

      </p>
      <p>
        Built with <a href="https://nene.leouieda.com">Nēnē</a> v0.5.0
        from source code at
        <a href="https://github.com/OllyK/nene_website"><code>OllyK/nene_website</code></a>
        (<a href="https://github.com/OllyK/nene_website/commit/840104b"><code>840104b</code></a>).
      </p>
      <p>
        Found a typo/mistake?
        <a target="_blank" href="https://github.com/OllyK/nene_website/edit/main/publications/index.md">
          Send a fix through GitHub
          <i class="fa fa-code-branch" aria-hidden="true"></i>
        </a>
      </p>
    </div>
  </footer>

  <!-- Bootstrap -->
  <script src="/pkg/bootstrap-5.1/bootstrap.bundle.min.js" integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ" crossorigin="anonymous"></script>
  <!-- KaTeX -->
  <script defer src="/pkg/katex-0.15.3/katex.min.js" integrity="sha384-0fdwu/T/EQMsQlrHCCHoH10pkPLlKA1jL5dFyUOvB3lfeT2540/2g6YgSi2BL14p" crossorigin="anonymous"></script>
  <script defer src="/pkg/katex-0.15.3/auto-render.min.js" integrity="sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR" crossorigin="anonymous"></script>
  <script>
    document.addEventListener("DOMContentLoaded", function() {
      renderMathInElement(document.body, {
        delimiters: [
          {left: '$$', right: '$$', display: true},
          {left: '$', right: '$', display: false},
        ],
      });
    });
  </script>
  <!-- Dimensions badge -->
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

</body>
</html>